program.c

```diff
@@ -0,0 +1,78 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+#include<string.h>
+int isOperator(char ch){
+    if(ch=='+' || ch=='-' || ch=='/' || ch=='*')
+        return 1;
+    else    
+        return 0;
+}
+int solve(int a,int b,char op){

```

Comment:
function names like **solve** should be more descriptive

Reply:
Function names are made descriptive in calculator branch.